### PR TITLE
UI-3219: Bump Footable to 3.1.6

### PR DESCRIPTION
**Pull request by guillegr123**

### CODE REVIEW

File: docs/handlebars.md [hunk 1]
```diff
@@ -357,3 +358,11 @@ example.html
 
 [i18n]: internationalization.md
 [i18n_templates]: internationalization.md#in-html-templates
+
+
+#### coalesce
```
[the remark this answers is not in the document]
Comment (guillegr123) [reply docs/handlebars.md]: Kinda, hehehe, I've seen it on SQL Server, but maybe it is from the SQL standard. The if was too verbose :(

File: src/js/lib/monster.ui.js [hunk 2]
```diff
@@ -19,6 +19,19 @@ define(function(require) {
...
+					throw new Error('Handlerbars Helper "coalesce" needs at least 2 parameters');
+				}
+
+				for (var i = 0; i < arguments.length - 1; i++) {
+					if (!_.isNil(arguments[i])) {
+						return arguments[i];
+					}
+				}
+				return null;
+			},
```
[the remark this answers is not in the document]
Comment (guillegr123) [reply src/js/lib/monster.ui.js]: Sorry, I forgot about it. Just remembered the part about not using `arguments` as a variable or parameter name. I'll change it.

[the remark this answers is not in the document]
Comment (guillegr123) [reply src/js/lib/monster.ui.js]: Resolved :+1:

File: src/apps/common/submodules/portListing/views/listing.html [hunk 3]
```diff
@@ -33,7 +33,7 @@
 			<tbody>
 			{{#each allOtherPorts}}
 				<tr class="listing-item" data-id="{{id}}">
-					<td>
+					<td data-sort-value="{{name}}{{#if reference_number}} {{replaceVar @root.i18n.portListing.listing.tbody.reference reference_number}}{{/if}}">
```
[the remark this answers is not in the document]
Comment (guillegr123) [reply src/apps/common/submodules/portListing/views/listing.html]: Resolved :+1:

File: src/apps/common/submodules/portListing/views/listing.html [hunk 4]
```diff
@@ -171,7 +167,7 @@ <h4 class="title">
 						</div>
 					{{/if}}
 					</td>
-					<td>
+					<td data-sort-value="{{@root.i18n.portListing.misc.status.unconfirmed}}">
```
[the remark this answers is not in the document]
Comment (guillegr123) [reply src/apps/common/submodules/portListing/views/listing.html]: Resolved :+1:

**Comment by azefiel** [comment]
I took the opportunity to clean up a bit how the data is sent to all ledger tables in `myaccount/balance`.

**Comment by azefiel** [comment]
Don't worry about this PR, I'll merge it when the time comes. Just want to make sure all the related PRs are ready to be merged in their respective repos.

**Pull request merged**